core/templates/pages/partnerships-page/partnerships-page.component.ts

```diff
@@ -90,17 +90,17 @@ export class PartnershipsPageComponent implements OnInit, OnDestroy {
     this.partneringImgUrl = this.urlInterpolationService.getStaticImageUrl(
       '/general/partnering_image.png');
     this.org1Url = this.urlInterpolationService.getStaticImageUrl(
-      '/general/org1.png');
+      '/partner_logos/movimentoAmplia.png');
     this.org2Url = this.urlInterpolationService.getStaticImageUrl(
-      '/general/org2.png');
+      '/partner_logos/digitalCitizen.png');
     this.org3Url = this.urlInterpolationService.getStaticImageUrl(
-      '/general/org3.png');
+      '/partner_logos/injazPalestine.png');
     this.org4Url = this.urlInterpolationService.getStaticImageUrl(
-      '/general/org4.png');
+      '/partner_logos/nairoBits.png');
```

Comment:
"nairobits" is one word (so please don't capitalize "B"), but everything else seems good. Thanks!

Reply:
Thanks for feedback
PTAL again